databases/week2/online-db.sql

```diff
@@ -0,0 +1,75 @@
+CREATE DATABASE IF NOT EXISTS `online-db`;
+USE `online-db`;
+DROP DATABASE IF EXISTS `online-db`;
+
+DESCRIBE customer;
+DROP TABLE customer;
+CREATE TABLE customer(
+id int unsigned NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR (255) NOT NULL,
+  last_name VARCHAR (255) NOT NULL,
+  phone VARCHAR (255) NOT NULL,
+  email VARCHAR (255) NOT NULL,
+  PRIMARY KEY (id)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+DESCRIBE product;
+DROP TABLE product;
+CREATE TABLE product(
+id int unsigned NOT NULL AUTO_INCREMENT,
+  product_name VARCHAR (255) NOT NULL,
+  product_price VARCHAR (255) NOT NULL,
+  cust_id INT UNSIGNED,
+  PRIMARY KEY (id),
+  CONSTRAINT `fk_product`
+  FOREIGN KEY (`cust_id`)
+  REFERENCES `online-db`.`customer` (`id`)
+  ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+
+DESCRIBE orders ;
+CREATE TABLE orders (
```

Comment:
Please correct me if i'm wrong, i might be missing something, but i'd say an order will have several products and will be associated to one costumer, but in your database one product is associated with one customer and then one customer is associated to one order (which i'm not sure i understand what it means in practice). Is this correct or am i missing something?
If i happen to be correct, i'd suggest having an order being constituted by multiple products and be associated to one customer. And since the same product can also be part of more than one order, you'll have a many to many relationship, which is a requirement of the homework :)

Reply:
I am missing many to many relationship right. I will try to fix it